pyiron_base/jobs/flex/executablecontainer.py

```diff
@@ -79,6 +79,9 @@ def write_input(self):
                 working_directory=self.working_directory,
             )
 
+    def run_static(self):
+        self.storage.output.command_line = super().run_static()
```

Comment:
I would call this `stdout` or something like this, because when I read `command_line` I'd expect the command that was run, otherwise looks good to me.

Reply:
I like the suggestion and renamed `command_line` to `stdout`.